src/index.css

```diff
@@ -9351,6 +9351,7 @@ a:hover {
 
 .text-mid-grey {
   color: #99a1b3;
+  overflow: auto;
```

Comment:
Thanks for this! Can you use the `.overflow-auto` class (https://tailwindcss.com/docs/overflow/#app) rather than edit the CSS file?

Reply:
Hey @jenkoian, my bad, I didn't realise you were using tailwind (should have read the [readme](https://github.com/jenkoian/hacktoberfest-checker#changing-styling)!)

Have updated.

Thanks!